docs/docs/get-started/get-started.md

```diff
@@ -79,4 +79,25 @@ kurtosis run github.com/kurtosis-tech/basic-service-package \
 
 ![service-c-partying.png](/img/home/service-c-k8s.png)
  
-</details>
\ No newline at end of file
+</details>
```

Comment:
@leeederek  I don't think this should go in the `get-started` section. its really important that we keep this as minimal as possible - strong opinion that we should keep this particular file the same, but include it in other places so its findable on the search bar.

Reply:
👍🏻 ok, makes sense. I've removed it.